src/python/pants/backend/jvm/tasks/jvm_compile/rsc/README.md

```diff
@@ -1,4 +1,8 @@
 Rsc Compile
 ===========
 
-Rsc is a new scala compiler that produces outlines instead of bytecode (https://github.com/twitter/rsc). This task uses that compiler and its toolchain to produce jars that scalac via zinc can compile against to produce bytecode containing jars.
\ No newline at end of file
+Rsc is a new scala compiler that produces outlines instead of bytecode (https://github.com/twitter/rsc). This task uses that compiler and its toolchain to produce jars that scalac via zinc can compile against to produce bytecode containing jars.
+
+In order to invoke `rsc`, you can invoke on a jvm target like:
+
+    $ ./pants --no-compile-{rsc,zinc}-incremental clean-all --compile-{rsc,zinc}-execution-strategy=hermetic --jvm-platform-compiler=rsc compile testprojects/src/scala/org/pantsbuild/testproject/mutual:bin  -ldebug
```

Comment:
I think it'd be a little easier to read this if the args were reordered to put the goals & target at the end  like
```suggestion
    $ ./pants -ldebug --no-compile-{rsc,zinc}-incremental --compile-{rsc,zinc}-execution-strategy=hermetic --jvm-platform-compiler=rsc clean-all compile testprojects/src/scala/org/pantsbuild/testproject/mutual:bin
```